extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/internal/lib/structure/devices/impl/JSONDeviceImpl.java

```diff
@@ -818,9 +851,41 @@ private boolean checkSceneNumber(Short sceneNumber) {
         }
     }
 
+    private Integer[] getStandartSceneOutput(short sceneNumber) {
+        switch (SceneEnum.getScene(sceneNumber)) {
+            case DEVICE_ON:
+            case MAXIMUM:
+                if (!isShade()) {
+                    return new Integer[] { (int) maxOutputValue, -1 };
+                } else {
+                    if (isBlind()) {
+                        return new Integer[] { (int) maxSlatPosition, (int) maxSlatAngle };
+                    } else {
+                        return new Integer[] { (int) maxSlatPosition, -1 };
+                    }
+                }
+                // off scenes
```

Comment:
At this time no break is missing, because every else part has a return argument and the break would be unreachable. By the next comment the same.